client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -143,14 +87,22 @@ const PluginsBrowser = ( { trackPageViews = true, category, search, hideHeader }
 				popularSearchesRef={ searchHeaderRef }
 				isSticky={ isAboveElement }
 				searchTerm={ search }
-				isSearching={ isFetchingPluginsBySearchTerm }
 				title={ translate( 'Plugins you need to get your projects done' ) }
 				searchTerms={ [ 'seo', 'pay', 'booking', 'ecommerce', 'newsletter' ] }
 			/>
 
-			{ ! search && <Categories selected={ category } /> }
-			<div className="plugins-browser__main-container">{ renderList() }</div>
-			{ ! category && ! search && <EducationFooter /> }
+			<Categories selected={ category } />
+			<div className="plugins-browser__main-container">
+				<PluginsDiscoveryPage
```

Comment:
Going by `PluginsSearchResultPage` and `PluginsCategoryResultsPage` I expect "Pages" to include a `MainComponent` root. Can we rename `PluginsDiscoveryPage` to something other than `Page`?